packages/admin-ui/src/pages/wifi/components/WifiStatus.tsx

```diff
@@ -0,0 +1,150 @@
+// React
+import React, { useEffect } from "react";
+import { useSelector } from "react-redux";
+import { NavLink } from "react-router-dom";
+// Own components
+import Card from "components/Card";
+import ErrorView from "components/ErrorView";
+import Loading from "components/Loading";
+import Switch from "components/Switch";
+import { confirm } from "components/ConfirmDialog";
+import { withToast } from "components/toast/Toast";
+// API
+import { api, useApi } from "api";
+import { continueIfCalleDisconnected } from "api/utils";
+import { getDappnodeIdentityClean } from "services/dappnodeStatus/selectors";
+// Types
+import { ContainerState, WifiReport } from "types";
+// css
+import "./wifi.scss";
+import { StateBadgeContainer } from "pages/packages/components/StateBadge";
+import { MdWifi } from "react-icons/md";
+// Utils
+import { prettyDnpName } from "utils/format";
+
+const wifiDnpName = "wifi.dnp.dappnode.eth";
+
+function WifiInfo({ wifiStatus }: { wifiStatus: ContainerState }) {
+  const dappnodeIdentity = useSelector(getDappnodeIdentityClean);
+  return (
+    <p>
+      Connect to the Wi-Fi hostpot exposed by your DAppNode using your{" "}
+      <a href="http://my.dappnode/#/wifi/credentials">credentials</a>. More
```

Comment:
Never write links directly. All links should be variables and declared in the links section of params https://github.com/dappnode/DNP_DAPPMANAGER/blob/6838be01fbd24e346c8c6e8c6a2a0940afcd939e/packages/admin-ui/src/params.ts#L65